app/furniture/marketplace/marketplaces_controller.rb

```diff
@@ -18,7 +18,6 @@ def update
           flash.notice = "Square notification settings updated succesfully!"
         end
       else
-        flash.alert = "Square notification settings were not upated. Please try again or contact your site administrator."
```

Comment:
Oops, didn't mean to leave this in here. Not too concerned yet with how we handle failure.